drivers/eks/eks_driver.go

```diff
@@ -864,6 +864,10 @@ func (d *Driver) waitForClusterReady(svc *eks.EKS, state state) (*eks.DescribeCl
 		}
 
 		status = *cluster.Cluster.Status
+
+		if status == eks.ClusterStatusFailed {
+			return nil, fmt.Errorf("cluster creation failed - cluster information: %s", cluster.Cluster.String())
```

Comment:
Let's not dump the whole cluster, just the name and ARN would allow a user to cleanup the cluster in AWS if needed.

Reply:
Indeed, the error message was weird. I have updated it to include only the name and ARN following your suggestion.